.env.default

```diff
@@ -43,3 +43,9 @@ STORAGE_KEY=ds
 
 # what to store, set empty or null to store all state
 CACHE_STATE_KEYS=["resource.session"]
+
+#Sentry configs
+#SENTRY_ORG=fe_sceleton
+#SENTRY_PROJECT=react
+#SENTRY_AUTH_TOKEN=4e03c798977a4c25b2168abbb8ad5d632b777e9bde934db3a9f6574b6403ef48
+#SENTRY_DSN = https://8013d8c0a75143728ceb60ea451d3ac8@sentry.io/1532358
```

Comment:
1. we only need DSN and ENVIRONMENT options here

2. please always set default values (empty value is allowed):
```
SENTRY_DSN=
SENTRY_ENVIRONMENT=dev
```